components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1688,13 +1702,14 @@ public Set<AccessTokenDO> getAccessTokensByUserForOpenidScope(AuthenticatedUser
             ps.setString(3, OAuthConstants.TokenStates.TOKEN_STATE_ACTIVE);
             ps.setString(4, authenticatedUser.getUserStoreDomain());
             ps.setString(5, OAuthConstants.Scope.OPENID);
+            ps.setTimestamp(6, new Timestamp(System.currentTimeMillis() + OAuth2Util.timestampSkew),
```

Comment:
Shall we use `OAuthServerConfiguration.getInstance().getTimeStampSkewInSeconds() * 1000` instead of `OAuth2Util.timestampSkew` ?

Reply:
Nonstatic + calculated/derived `OAuthServerConfiguration.getInstance().getTimeStampSkewInSeconds() * 1000` is costly when compared to already available static `OAuth2Util.timestampSkew`. Therefore, there was no other alternative.